src/components/FloatingLabel/FloatingLabel.tsx

```diff
@@ -4,12 +4,30 @@ import { twMerge } from 'tailwind-merge';
 import { mergeDeep } from '../../helpers/merge-deep';
 import { getTheme } from '../../theme-store';
 import type { DeepPartial } from '../../types';
-import type { FlowbiteSizes } from '../Flowbite';
+import type { FlowbiteColors, FlowbiteSizes } from '../Flowbite';
+
+export interface FlowbiteFloatingLabelHelperText extends Partial<FlowbiteColors> {
+  default: string
+  success: string
+  error: string
+}
 
 export interface FlowbiteFloatingLabelTheme {
-  input: any;
-  label: any;
-  helperText: any;
+  input: {
+    [key: string]: {
```

Comment:
I think that you could have created a type for this one since it is used in the +1 place